packages/windows/install-mroonga.ps1

```diff
@@ -20,13 +20,24 @@ function Wait-UntilTerminate($cmdName) {
 }
 
 function Install-Mroonga($mariadbVer, $arch, $installSqlDir) {
+  Write-Output("Start to install Mroonga")
   cd "mariadb-$mariadbVer-$arch"
+  if ("$mariadbVer" -eq "10.4.7") {
+    Write-Output("Clean data directory")
+    Remove-Item data -Recurse
+    Start-Process .\bin\mysql_install_db.exe
+    Start-Sleep -s 10
+  }
+  Write-Output("Start mysqld.exe")
   Start-Process .\bin\mysqld.exe
-  Wait-UntilRunning mysqld

```

Comment:
Why do we just run `Start-Sleep -s 10` instead of fixing this function?

Reply:
We can use `mysqladmin ping` to confirm that `mysqld` is ready: https://github.com/mroonga/docker/blob/master/test/build.sh#L25